index.js

```diff
@@ -17,8 +17,19 @@ const manageRecipes = async () => {
 
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
-
-    // Run your code here, after you have insured that the connection was made
+  
+    await Recipe.create({title:'FishFingers'});
+
+    await Recipe.insertMany (data);
+
+    let message = await Recipe.findOneAndUpdate({ title: 'Rigatoni alla Genovese' }, {duration: 100}, {new:true});
+    console.log (`Change successful:` + message.duration);
+   
+    let deleteRecipe = await Recipe.deleteOne({ title: 'Carrot Cake' }, {new:true});
```

Comment:
You don't need to set the `new` option to `true` here. It's used with the update function and returns the document after the update was applied.